src/main/java/com/parasoft/demoapp/service/OrderMQService.java

```diff
@@ -19,6 +25,13 @@ public class OrderMQService {
     @Autowired
     private JmsMessagingTemplate jmsMessagingTemplate;
 
+    @Autowired
+    private KafkaTemplate operationRequestKafkaTemplate;
+
+    @Lazy
+    @Autowired
+    private GlobalPreferencesService globalPreferencesService;
```

Comment:
No need to introduce globalPreferencesService, we can get mqType by `MQConfig.currentMqType`.